internal/infrastructure/storage/db/pg/migration/20230504102232_init.up.sql

```diff
@@ -0,0 +1,63 @@
+CREATE TABLE market (
+    name VARCHAR(50) NOT NULL PRIMARY KEY,
+    base_asset VARCHAR(255) NOT NULL,
+    quote_asset VARCHAR(255) NOT NULL,
+    base_asset_precision INTEGER NOT NULL DEFAULT 8,
+    quote_asset_precision INTEGER NOT NULL DEFAULT 8,
+    tradeable BOOLEAN NOT NULL DEFAULT FALSE,
```

Comment:
This is not really a typo, but the field at domain level is called Tradable, so let's stick with it.

```suggestion
    tradable BOOLEAN NOT NULL DEFAULT FALSE,
```